### PR TITLE
Redo tasks

### PR DESCRIPTION
**Pull request by demkantor**
workflow now looks more like client side

### CODE REVIEW

**Comment by haley-r** [comment]
Hey David, can you summarize what's happening here? Or maybe break down important changes vs cleanup/minor edits? Trust you that it works but would like to know what's going on!

**Comment by meghen** [comment]
hey so i noticed in checking this branch against the master --
it looks like after creating a new workflow in master, the user is automatically routed to keep working on building that workflow. in this branch, it sweet alerts but then puts you back to the 'create workflow (name and description) page. that project is still showing up in existing workflows, but its not the most user friendly to not route them there.

**Comment by demkantor** [comment]
most of the changes is in removing the side navbar and making it feel more like the subscriber side layout. there were more ui/ux changes than anything. to address the jumping back after creating a new workflow that probably isnt best, ill move the user back into continuing workflow creation instead. let me know if other changes are needed!
